control_panel_api/tests/test_aws.py

```diff
@@ -0,0 +1,65 @@
+from unittest.case import TestCase
+from unittest.mock import patch
+
+from control_panel_api import aws
+
+
+@patch('boto3.client')
+class AwsTestCase(TestCase):
+    def test_create_bucket(self, mock_client):
+        aws.create_bucket('bucketname', 'eu-west-1')
+        mock_client.return_value.create_bucket.assert_called()
```

Comment:
Can't remember how to assert a method was called with certain params. 
Do you think it may be worth to be a bit more specific in some of these tests?

Reply:
I've tried to not make the tests too granular and brittle if not necessary.  These are basic parameter passing no real domain logic in them.

Reply:
I see, so depending on the thing you're testing and it's probability to write it wrong you write more or less detailed tests?

OK, I would probably be more specific when there are possible security implications at the cost of making tests more brittle (e.g. policy document).